tests/model_serving/model_server/kserve/negative/test_invalid_inference_requests.py

```
@@ -0,0 +1,141 @@
+    }
+
+    @pytest.mark.parametrize(
+        "content_type",
+        [
+            pytest.param("text/xml", id="text_xml"),
+            pytest.param("application/x-www-form-urlencoded", id="form_urlencoded"),
+        ],
+    )
+    def test_unsupported_content_type_returns_415(
+        self,
+        unprivileged_client: Any,
```

Ruff (0.15.1)

[warning] Unused method argument: `unprivileged_client`

(ARG002)